packages/api/src/middleware/hardcoded-nodes.test.ts

```diff
@@ -76,7 +76,7 @@ describe("kubernetes middleware", () => {
   });
 
   it("should return orchestrators from getOrchestrators()", async () => {
-    const response = await req.getOrchestrators();
+    const response = await req.orchestratorsGetters[0]();
```

Comment:
Could we keep calling the `getOrchestrators` function that iterates over the getters list?